client/js/views/GameView.vue

```diff
@@ -209,6 +210,7 @@
       </div>
       <!-- Player Hand -->
       <div id="player-hand" class="d-flex flex-column justify-end align-center px-2 pt-2 mx-auto">
+        <h4 data-cy="player-username">{{ playerUsername }}</h4>
```

Comment:
This puts the player's username above the goal + points, right? Do you have an uncommitted change that moves it down below?